Resources/ServerInfo/Guidebook/_RMC14/Law/RMCMarineLawSearches.xml

```diff
@@ -0,0 +1,28 @@
+﻿<Document>
+  # Searches
+
+  - Searches must be approved by the [color=#8B0000][bold]Chief MP[/bold][/color] or [color=#008000][bold]Commander[/bold][/color].
+  - If waiting would cause further harm to the ship or personnel, you may gain approval after performing the search.
+  - Searches can be performed by the [color=#008000][bold]Commander[/bold][/color] or any [color=#ff000d][bold]MP[/bold][/color], as long as the following procedure is followed.
+  - If procedure is not followed, the offender can be charged with [bold]Neglect of Duty[/bold].
+
+  ## Personnel
+  - Inform the person you are going to conduct a search and the reason for the search.
+  - Use non-lethal force to restrain them if they do not comply.
+  - Where possible, conduct the search in the brig or any isolated area so nothing is stolen.
+  - Begin the search, confiscate any illegal items, and secure them.
+  - If the items warrant a sentence, follow arrest and brig procedures, otherwise release the personnel. Searches cannot incur a [bold]Prevarication[/bold] charge
```

Comment:
"Searches cannot incur a [bold]Prevarication[/bold] charge" is missing a period at the end of the sentence and feels out of place. This, if I understand it correctly, should be placed in the general section on searches. A more clear wording such as "Performing a search cannot ..." would be preferable too.